wiki/People/Beatmap_Nominators/en.md

```diff
@@ -43,119 +41,147 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
-| [Sotarks](https://osu.ppy.sh/users/4452992)           | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | French                   |
-| [squirrelpascals](https://osu.ppy.sh/users/6151332)   | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                          |
-| [Stefan](https://osu.ppy.sh/users/626907)             | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | German                   |
-| [Strategas](https://osu.ppy.sh/users/2971837)         | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Lithuanian               |
-| [Surono](https://osu.ppy.sh/users/3611370)            | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | Indonesian               |
-| [tasuke912](https://osu.ppy.sh/users/2774767)         | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | Japanese                 |
-| [TheKingHenry](https://osu.ppy.sh/users/5128277)      | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Finnish                  |
-| [timemon](https://osu.ppy.sh/users/2072005)           | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Thai                     |
-| [Tofu1222](https://osu.ppy.sh/users/6089608)          | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] | Chinese                  |
-| [UndeadCapulet](https://osu.ppy.sh/users/2523533)     | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                          |
-| [Voli](https://osu.ppy.sh/users/2522275)              | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                          |
-| [Volta](https://osu.ppy.sh/users/4154071)             | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | Indonesian               |
-| [Wafu](https://osu.ppy.sh/users/888955)               | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Czech                    |
-| [Weber](https://osu.ppy.sh/users/6410432)             | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] |                          |
-| [Wishkey](https://osu.ppy.sh/users/1101704)           | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Dutch, German            |
-| [xfraczynho](https://osu.ppy.sh/users/2192669)        | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | Spanish                  |
-| [YaHao](https://osu.ppy.sh/users/2633753)             | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] | Chinese                  |
-| [ZiRoX](https://osu.ppy.sh/users/200768)              | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] | Spanish                  |
+The tables listed below indicate the game mode(s) in which each Beatmap Nominator is formally qualified to nominate.
+_Please note: All BN speak English unless otherwise noted._ Link to the [user group page](https://osu.ppy.sh/groups/28).
```

Comment:
prepend an empty line or merge this line with 61

Reply:
done